.buildkite/steps/cleanup.sh

```diff
@@ -4,8 +4,7 @@ set -uxo pipefail
 
 ########################################################
 # We make an attempt to clean up the specific resources created during this pipeline elsewhere. However
-# sometimes that fails, are various CloudFormation Stacks, S3 Buckets and CloudWatch Log Groups can be
-# left around.
+# sometimes that fails, are various CloudFormation Stacks and S3 Buckets left around.
```

Comment:
Should "are various CloudFormation Stacks and S3 Buckets left around." say "and various CloudFormation Stacks and S3 Buckets are left around."?